sql/moz-fx-data-shared-prod/stripe_external/nonprod_discount_v1/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+friendly_name: Non-production Stripe discounts
+description: |-
+  Mirror of non-production Stripe `discount` table from Fivetran.
+owners:
+- srose@mozilla.com
+labels:
+  incremental: false
+  schedule: hourly
+scheduling:
+  dag_name: bqetl_cjms_nonprod
+  # The whole table is overwritten every time, not a specific date partition.
+  date_partition_parameter: null
+  depends_on_fivetran:
+  - task_id: fivetran_stripe_nonprod
+bigquery:
+  time_partitioning: null
+  clustering: null
+workgroup_access:
+- role: roles/bigquery.dataViewer
+  members:
+  - workgroup:mozilla-confidential
```

Comment:
This [flagged DSRE as granting mozilla-confidential access in an otherwise restricted access dataset](https://mozilla.slack.com/archives/CF32SJCDS/p1729822304902239). I'm assuming that's desirable given other tables don't have this set but if it's accidental copypasta you can remove said access by removing `workgroup_access` in a followup PR.

Reply:
That was indeed intentional.  All the `stripe_external.nonprod_*` ETLs are configured with Moz-confidential access, so I was following that convention for this new ETL as well.